### ISSUE TITLE
[Bug]: In some situations Playwright Webkit is unable to find elements when SVG is used as a background with "background-size CSS applied

### ISSUE THREAD

**Issue by RichardCariven** [microsoft/playwright#32783]
### Version

1.47.2

### Steps to reproduce

If the css is applied as follows 

```
</> HTML
<div className="background">
    <Nav />
    <Main />
    <Footer />
</div>
```

```
</> CSS
.background {
    background-repeat: no-repeat;
    background-image: url(bg.svg);
    background-repeat: no-repeat;
    background-position: center top -170px;
    background-size: 2300px auto; //This makes webkit tests fail
}
```

And the test is run on CI/CD as a Github action (might also be the case with other pipelines) with the following set up 

Github Action
runs-on: ubuntu-24.04
image: mcr.microsoft.com/playwright:v1.47.2
@playwright/test version 1.47.2

Playwright Config
```
projects: [
    {
      grep: [/@web/, /@all/],
      name: "Chrome-Desktop",
      use: { ...devices["Desktop Chrome"] },
    },
    {
      grep: [/@web/, /@all/],
      name: "Safari-Desktop",
      use: { ...devices["Desktop Safari"] },
    },
```

Then on the Webkit runs the svg Background image seems to make some elements on the centre section of the page unreachable to Playwright (locators show as Hidden). When then checking the Visibility of the element 

`await expect(elementIdentifier).toBeVisible();`

Playwright shows the element as Hidden, On Chrome Playwright can find the element and complete the test as expected. 

This only seems to occur on the CI/CD Pipeline, running the tests locally Playwright runs as expected. 

In order to "fix" the issue devs removed the Background Size. 

### Expected behavior

I would expect Playwright to find the elements on Webkit in exactly the same way as it did for Chrome and when running the tests locally. 

### Actual behavior

playwright is unable to "see" elements seemingly behind the SVG while it has a background size applied. 

### Environment

```shell
Local output tests passing  

System:
    OS: macOS 14.7
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 3.61 GB / 32.00 GB
  Binaries:
    Node: 18.17.0 - ~/.nvm/versions/node/v18.17.0/bin/node
    Yarn: 1.22.22 - ~/.yarn/bin/yarn
    npm: 10.8.1 - ~/.nvm/versions/node/v18.17.0/bin/npm
  Languages:
    Bash: 3.2.57 - /bin/bash

Environment the issue is taking place 

Github Action
runs-on: ubuntu-24.04
image: mcr.microsoft.com/playwright:v1.47.2
@playwright/test version 1.47.2
```

**Comment by yury-s** [comment]
We need a complete minimal example that we could run locally and reproduce the issue. Can you follow our bug template and share such repro?

**Comment by RichardCariven** [comment]
Will see what I can do, however might be tricky as requires the App code be shared alongside the test code.

**Comment by yury-s** [comment]
It will likely fail on Linux without docker too but we need a repro before we can take any action on your report.

**Comment by RichardCariven** [comment]
I don't have a linux machine, and want to make sure I can recreate the failure first, might be a week or 2 before I get the bandwidth to make the static page that matches the page in APP causing the failure (not a dev and the devs def don't have bandwidth to do more then code review what I come up with)

**Comment by yury-s** [comment]
Please file a new issue with a repro when you have time. I'll close this report for now as part of triaging process, we cannot do anything about it without a reproduction anyway.
